src/main/kotlin/net/atos/client/smartdocuments/SmartDocumentsClient.kt

```diff
@@ -47,4 +41,12 @@ interface SmartDocumentsClient {
         @HeaderParam("Authorization") authenticationToken: String,
         @HeaderParam("Username") userName: String
     ): SmartDocumentsTemplatesResponse
+
+    @GET
+    @Path("smartdocuments/result/show")
+    @Consumes("text/html; charset=UTF-8")
+    fun downloadFile(
+        @QueryParam("id") smartDocumentsId: String,
```

Comment:
`smartDocumentsId` sounds a bit weird to me? Is it not just a:

```suggestion
        @QueryParam("id") fileId: String,
```

?